app/src/main/res/layout/fragment_criterions.xml

```diff
@@ -1,145 +1,156 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="20dp"
+        android:paddingEnd="20dp"
+        android:paddingTop="20dp"
+        android:paddingBottom="20dp"
+        android:orientation="vertical"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/criterions_title"
+            android:fontFamily="@font/fredoka_bolder"
+            android:textColor="@color/theme_dark"
+            android:textSize="24sp"
+
+            android:gravity="center"
+            android:layout_marginBottom="18dp" />
+
+        <TextView
+            android:id="@+id/criterions_age_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:fontFamily="@font/fredoka_normal"
+            android:gravity="center"
+            android:text="@string/criterions_age_title"
+            android:textSize="20sp" />
+
+        <TextView
+            android:id="@+id/criterions_age_desc"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:alpha="0.6"
+            android:fontFamily="@font/fredoka_normal"
+            android:gravity="center"
+            android:text="@string/criterions_age_desc"
+            android:textSize="14sp" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:gravity="center">
+
+            <TextView
+                android:id="@+id/minValueAge"
+                android:layout_width="45dp"
+                android:gravity="right"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/fredoka_normal"
+                android:text="@string/criterions_age_min"
+                android:textSize="20sp" />
+
+            <com.google.android.material.slider.RangeSlider
+                android:id="@+id/rangeAgeSelector"
+                android:stepSize="1.0"
+                android:layout_weight="1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="10dp"
+                android:layout_marginEnd="10dp"
+                android:valueFrom="13"
```

Comment:
MAGIC NUMBER ALERT !!

Reply:
Yes, could replace as well the margins and other numbers with stored value so it keeps consistent throughout the app

Reply:
valueFrom and valueTo will be changed, for the margin it could be an open internal feature refactoring that could be considered